Formula/habitat.rb

```diff
@@ -0,0 +1,15 @@
+class Habitat < Formula
+  desc "Application automation framework to build applications"
+  homepage "https://www.habitat.sh/"
+
+  url "https://api.bintray.com/content/habitat/stable/darwin/x86_64/hab-0.6.0-20160613151520-x86_64-darwin.zip?bt_package=hab-x86_64-darwin"
```

Comment:
Thanks for the information, I had trouble using the `$latest` syntax and just replaced with the name of the file as I didn't know the alternate URL